collector/entropy_linux.go

```diff
@@ -33,23 +36,44 @@ func init() {
 
 // NewEntropyCollector returns a new Collector exposing entropy stats.
 func NewEntropyCollector(logger log.Logger) (Collector, error) {
+	fs, err := procfs.NewFS(*procPath)
+	if err != nil {
+		return nil, fmt.Errorf("failed to open procfs: %w", err)
+	}
+
 	return &entropyCollector{
+		fs: fs,
 		entropyAvail: prometheus.NewDesc(
 			prometheus.BuildFQName(namespace, "", "entropy_available_bits"),
 			"Bits of available entropy.",
 			nil, nil,
 		),
+		entropyPoolSize: prometheus.NewDesc(
+			prometheus.BuildFQName(namespace, "", "entropy_pool_size_bytes"),
```

Comment:
Why are we mixing bits and bytes here?

Reply:
It's bits for kernel 2.6+ and bytes for 2.4, that's why i got confused, will change for bits.

Reply:
We don't support kernels older than 2.6.23, because Go will not compile on older kernels.